sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -910,4 +910,20 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("SPARK-18217: Disallow altering permanent views based on temporary views") {
```

Comment:
let's move this to `PersistedViewTestSuite`

Reply:
and let's also fix the jira number

Reply:
OK